components/modal/modal.css

```diff
@@ -0,0 +1,41 @@
+.class-modal {
+    display: none; 
+    position: fixed; 
+    z-index: 2; 
+    left:15rem;
+    top:0;
+    width: 75%; 
+    height: 75%; 
+    transition: all 0.3s ease-in-out;
+  }
+  
+  .modal-content {
+    background-color: var(--secondary-color);
+    margin: 15% auto; /* 15% from the top and centered */
+    padding: 20px;
+    border: 1px solid var(--color-gray);
```

Comment:
try to keep the variable names in the same pattern - like `--color-gray and --color-secondary `